Ports/zsh/package.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=zsh
+version=zsh-5.8
+files="https://github.com/zsh-users/zsh/archive/refs/tags/${version}.zip zsh-${version}.zip 9fb194253336ae5871913a583e2bdb4db117bed80700539b2524cff606c8ff6f"
```

Comment:
This seems to use an unofficial mirror for the port. Is this intentional?

Reply:
I don't know how to download the source consistently via SourceForge. Any help would be appreciated ;)